Model/Menu/Node.php

```diff
@@ -14,8 +14,284 @@ protected function _construct()
         $this->_init('Snowdog\Menu\Model\ResourceModel\Menu\Node');
     }
 
+    /**
+     * @return array|string[]
+     */
     public function getIdentities()
     {
         return [self::CACHE_TAG . '_' . $this->getId()];
     }
+
+    /**
+     * Get node id
+     *
+     * @return int
+     */
```

Comment:
Same here, I think `@inheritdoc` is a better approach

Reply:
updated